keyboards/frobiac/blackbowl/readme.md

```diff
@@ -0,0 +1,36 @@
+# frobiac/blackbowl
+
+![frobiac/blackbowl](https://i.imgur.com/nehpp3f.jpeg)
```

Comment:
```suggestion
![frobiac/blackbowl](https://i.imgur.com/nehpp3fh.jpeg)
```

Reply:
Thanks, missed that one also in redtilt/